commands/metal_utils.py

```
@@ -0,0 +1,224 @@
+"""
+metal_utils.py
+
+このスクリプトは、Metal GPUを使った画像処理機能を提供します。
+Apple Silicon(M1/M2/M3)チップでのGPU処理を最適化し、
+CPU使用率を軽減して処理速度を向上させます。
+"""
+
+import os
+import numpy as np
+import cv2
+from typing import Tuple, List, Optional
```

Ruff (0.8.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)